### DISCUSSION TITLE
Let me pick a filename when exporting functions

### DISCUSSION THREAD

**Discussion by Caeylinity** [NilsSve/DfRefactor#26] [Ideas]
I just wanted to test the Import/Export functionality, so I wanted to make a backup JSON file containing all functions and a file containing only a couple functions, but I have no way to set a filename so the newest export overwrote the older export.

Instead of having to manually navigate to the file and rename it before making the second export, it would be nice if I could set a filename before exporting.

**Comment by NilsSve** [comment]
This has been fixed. Both an Export and the Import file can be selected and is automatically saved to the user's registry.
